src/editing.md

```diff
@@ -401,6 +401,10 @@ The inner cloze is entirely nested within the outer. There is no support for par
 
 with the word "was" appearing in both deletions.
 
+In version 24.11 nested cloze deletions have been limited to a maximum of three levels.
+This change had been introduced in [PR #3264](https://github.com/ankitects/anki/pull/3264) to prevent
```

Comment:
Not sure about this. The docs don't need to mention the pull request as it's mainly meant for end-user consumption. We don't do this anywhere else.

Reply:
Yes, that's fine.